src/amo/pages/Addon/index.js

```diff
@@ -467,6 +467,16 @@ export class AddonBase extends React.Component {
     }
   }
 
+  renderJsonLinkedData() {
+    const { addon } = this.props;
+
+    return (
+      <script type="application/ld+json">
+        {JSON.stringify(getAddonJsonLinkedData({ addon }))}
```

Comment:
since you have this this could return null should we wrap this in a condition?

Reply:
Actually, it should not return `null`, good catch